docs/guides/custom-publisher-integration.md

```diff
@@ -29,8 +29,8 @@ The following are the options available for publishers to integrate with UID2:
 
 - Client UID2 SDK for JavaScript (see [UID2 SDK for JavaScript Reference Guide](../sdks/client-side-identity.md)), with [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) on the server.
 - Client UID2 SDK for JavaScript (see [UID2 SDK for JavaScript Reference Guide](../sdks/client-side-identity.md)), with custom server code.
-- Server-only integration, with [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) or [UID2 SDK for Python](../sdks/uid2-sdk-ref-python.md) on the server.
-- Server-only integration, with custom server code.
+- Client-server integration, with [UID2 SDK for Java](../sdks/uid2-sdk-ref-java.md) or [UID2 SDK for Python](../sdks/uid2-sdk-ref-python.md) on the server.
+- Server-side integration, with custom server code.
 
 This guide provides information for the last two options.
```

Comment:
I think this guide now only provides information for the last option. If they're using a client-server integration (the second-last option), they'd be looking at the client-server guide.

Reply:
I think we can replace the whole paragraph with:

_See also [FAQs](http://localhost:3006/docs/guides/custom-publisher-integration#faqs).

This guide provides information for doing server-side integration using [UID2 SDK for Java](http://localhost:3006/docs/sdks/uid2-sdk-ref-java) or [UID2 SDK for Python](http://localhost:3006/docs/sdks/uid2-sdk-ref-python) or custom server codes, 

There is also an example application that demonstrates the workflow. See [Sample Application](http://localhost:3006/docs/guides/custom-publisher-integration#sample-application)._